dev/com.ibm.ws.request.probe.audit.rest/src/io/openliberty/request/probe/audit/rest/service/internal/AuditPERestServiceImpl.java

```diff
@@ -0,0 +1,84 @@
+package io.openliberty.request.probe.audit.rest.service.internal;
+
+import java.util.Map;
+import java.util.List;
+import java.util.concurrent.ConcurrentHashMap;
+
+import com.ibm.websphere.ras.Tr;
+import com.ibm.websphere.ras.TraceComponent;
+import com.ibm.websphere.security.audit.AuditConstants;
+import io.openliberty.security.audit.source.rest.MemberManagementEvent;
+import io.openliberty.security.audit.source.rest.RESTAuthorizationEvent;
+import com.ibm.ws.security.audit.source.AuditServiceImpl;
+
+import org.osgi.service.component.ComponentContext;
+import org.osgi.service.component.annotations.ConfigurationPolicy;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Modified;
+import org.osgi.service.component.annotations.Deactivate;
+
+import com.ibm.wsspi.security.audit.AuditService;
+import com.ibm.wsspi.kernel.service.location.VariableRegistry;
+import com.ibm.wsspi.kernel.service.utils.AtomicServiceReference;
+
+import com.ibm.ws.request.probe.audit.servlet.AuditPERestService;
+
+@Component(service = AuditPERestService.class, configurationPolicy = ConfigurationPolicy.IGNORE,
+           property = "service.vendor=IBM", immediate = true)
+
+public class AuditPERestServiceImpl implements AuditPERestService {
+
+
+		private static final TraceComponent tc = Tr.register(AuditPERestServiceImpl.class);
```

Comment:
This file is still using tabs everywhere.  It really messes up the format and makes things wrap when viewing in github when it doesn't need to.

Reply:
Also you are registering a trace component, but never using it and you have instrumentation disabled so nothing will be injected either.  I suggested adding something to the activate if you want something to show things being activated for debug purposes.

Reply:
Removed the TraceComponent. Not needed.